components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsBaseGraphBuilder.java

```diff
@@ -198,6 +201,178 @@ private void addLongWaitProcessInternal(AsyncProcess asyncProcess,
 
     protected abstract Function<Object, SerializableJsFunction> effectiveFunctionSerializer();
 
+
+    /**
+     * Adds the step given by step ID tp the authentication graph.
```

Comment:
Adds the step given by step ID *to* the authentication graph.

Spelling mistake

Reply:
corrected